watertap/unit_models/osmotically_assisted_reverse_osmosis_base.py

```diff
@@ -0,0 +1,818 @@
+            var_dict["Feed-side Hydraulic Diameter"] = self.feed_side.dh
+
+        if self.config.has_full_reporting:
+            expr_dict["Average Solvent Flux (LMH)"] = (
+                self.flux_mass_phase_comp_avg[time_point, "Liq", "H2O"] * 3.6e3
+            )
+            if hasattr(self.feed_side, "N_Re_avg"):
+                expr_dict[
+                    "Average Feed-side Reynolds Number"
+                ] = self.feed_side.N_Re_avg[time_point]
+            for j in self.config.property_package.solute_set:
+                expr_dict[f"{j} Average Solute Flux (GMH)"] = (
+                    self.flux_mass_phase_comp_avg[time_point, "Liq", j] * 3.6e6
+                )
+                if hasattr(self.feed_side, "K_avg"):
+                    expr_dict[
+                        f"{j} Average Feed-side Mass Transfer Coefficient (mm/h)"
+                    ] = (self.feed_side.K_avg[time_point, j] * 3.6e6)
+
+        # TODO: add more vars
+        return {"vars": var_dict, "exprs": expr_dict}
+
+    # permeate properties need to rescale solute values by 100
+    def _rescale_permeate_variable(self, var, factor=100):
+        if var not in self._permeate_scaled_properties:
+            sf = iscale.get_scaling_factor(var)
+            iscale.set_scaling_factor(var, sf * factor)
+            self._permeate_scaled_properties.add(var)
+
+    def calculate_scaling_factors(self):
+
+        super().calculate_scaling_factors()
+
+        # these variables should have user input, if not there will be a warning
+        if iscale.get_scaling_factor(self.area) is None:
+            sf = iscale.get_scaling_factor(self.area, default=10, warning=True)
+            iscale.set_scaling_factor(self.area, sf)
+
+        if iscale.get_scaling_factor(self.A_comp) is None:
+            iscale.set_scaling_factor(self.A_comp, 1e12)
+
+        if iscale.get_scaling_factor(self.B_comp) is None:
+            iscale.set_scaling_factor(self.B_comp, 1e8)
+
+        if iscale.get_scaling_factor(self.recovery_vol_phase) is None:
+            iscale.set_scaling_factor(self.recovery_vol_phase, 1)
+
+        for (t, p, j), v in self.recovery_mass_phase_comp.items():
+            if j in self.config.property_package.solvent_set:
+                sf = 1
+            elif j in self.config.property_package.solute_set:
+                sf = 100
+            if iscale.get_scaling_factor(v) is None:
+                iscale.set_scaling_factor(v, sf)
+
+        for v in self.rejection_phase_comp.values():
+            if iscale.get_scaling_factor(v) is None:
+                iscale.set_scaling_factor(v, 1)
+
+        if hasattr(self, "structural_parameter"):
+            if iscale.get_scaling_factor(self.structural_parameter) is None:
+                # Structural parameter expected to be ~ 1200 microns
+                iscale.set_scaling_factor(self.structural_parameter, 1e3)
+        # if not hasattr(self, "_permeate_scaled_properties"):
```

Comment:
Is there any reason this comment block can't be removed?

Reply:
I cleaned this up.